contracts/schemes/CL4RRedeemer.sol

```diff
@@ -0,0 +1,141 @@
+pragma solidity ^0.6.12;
+// SPDX-License-Identifier: GPL-3.0
+
+import "./ContinuousLocking4Reputation.sol";
+
+/**
+ * @title A scheme for continuous locking ERC20 Token for reputation
```

Comment:
please add a short description of this contract .
why it is needed and how it is suppose to work.